Library/Formula/vault.rb

```diff
@@ -0,0 +1,77 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  url "https://github.com/hashicorp/vault.git",
+      :tag => "v0.1.2",
+      :revision => "aa55896b93ba8127f79c8aa7b5707868f085b6d1"
+
+  head "https://github.com/hashicorp/vault.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/tools/godep" do
+    url "https://github.com/tools/godep",
+        :revision => "e2d1eb1649515318386cc637d8996ab37d6baa5e", :using => :git
+  end
+
+  # godep's dependencies
+  go_resource "github.com/kr/fs" do
+    url "https://github.com/kr/fs",
+        :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b", :using => :git
+  end
+
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools",
+        :revision => "997b3545fd86c3a2d8e5fe6366174d7786e71278", :using => :git
+  end
+
+  go_resource "github.com/mitchellh/gox" do
+    url "https://github.com/mitchellh/gox",
+        :revision => "a5a468f84d74eb51ece602cb113edeb37167912f", :using => :git
+  end
+
+  # gox dependency
+  go_resource "github.com/mitchellh/iochan" do
+    url "https://github.com/mitchellh/iochan",
+        :revision => "87b45ffd0e9581375c491fef3d32130bb15c5bd7", :using => :git
+  end
+
+  def install
+    contents = Dir["{*,.git,.gitignore,.travis.yml}"]
+    gopath = buildpath/"gopath"
+    gopath.mkpath
+
+    target = gopath/"src/github.com/hashicorp/vault"
+    target.mkpath
+    mv contents, target
```

Comment:
You can merge the three lines above by writing `(gopath/"src/github.com/hashicorp/vault").install contents`.